### ISSUE TITLE
Problem with SSR: ReferenceError: window is not defined

### ISSUE THREAD

**Issue by zamirkhan** [calibreapp/react-live-chat-loader#8]
It seems that this module does not work in a browserless situation (e.g. server-side rendering). To workaround, I can conditionally render the component, but that's a bit of a pain. It would be nice if the module could detect whether the browser is present (and if not, there is no need to do anything).

**Comment by michaeldijkstra** [comment]
Hey @zamirkhan,

We're currently using this ourselves on a next.js app which is server side rendered, as is the [example application](https://github.com/calibreapp/react-live-chat-loader-example-app). What issue are you running into? Any additional context you can provide (or a link to an example repo) would be greatly appreciated.

Also, we'd be happy to accept a PR to improve this if you're interested in contributing to this library?

**Comment by zamirkhan** [comment]
Hi @michaeldijkstra, sorry for the lack of context initially (I think I just lazily assumed SSR wasn't addressed, as it is sometimes an after-thought).

For the react app I'm working on, I use this (https://github.com/antonybudianto/cra-universal) for SSR, since it started as a simple CRA and the need for SSR came later. This works fine. When I intergrated `react-live-chat-loader`, it seemed to work fine locally (where I can run a mock SSR setup), but when I deployed it, I was getting errors on references to the `window` object within this package's code. I tried to reproduce it locally just now, but cannot, and (since I have a working solution, that avoids loading `react-live-chat-loader` when running on the server), I unfortunately don't have time to go through a deploy cycle right now to find the eact line. I know that's not entirely helpful. I do see references to `window` in the code and I also see  a place in `useChat.js` where there does appear to be a check for whether the window object exists. I just don't know if that covers all the possible entry points for the code.

Anyway, I appreciate the improvement this package does provide - thank you.
